README.md

```diff
@@ -27,10 +27,10 @@ pip install topiary
 ```
 
 You'll need to download the reference genome sequences and annotations for a
-recent Ensembl release (e.g. 79) by running:
+recent Ensembl release (e.g. 81) by running:
 
 ```
-pyensembl install --release 79
+pyensembl install --release 81 --species human
```

Comment:
The default is still human, but I've started adding the "--species" flag to
documentation to make it clear that its omission is selecting human data.

On Mon, Aug 24, 2015 at 3:39 PM, Tavi Nathanson <notifications@github.com>
wrote:

> In README.md
> <https://github.com/hammerlab/topiary/pull/20#discussion_r37794410>:
>
> >
> >  ```
> > -pyensembl install --release 79
> > +pyensembl install --release 81 --species human
>
> Remind me: do we default to species human or species is now required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/topiary/pull/20/files#r37794410>.
>